README.md

```diff
@@ -22,41 +18,26 @@ Follow the steps below to get this application fully functional and running usin
 
 1. Create a [Stytch](https://stytch.com/) account. Once your account is set up a Project called "My first project" will be automatically created for you.
 
-2. Within your new Project, navigate to [SDK configuration](https://stytch.com/dashboard/sdk-configuration), and make the following changes:
-
-   - Under **Test environment**, Click **Enable SDK in test**.
-   - Under **Authorized applications** add the domain `http://localhost:3000` if it isn't already there.
-   - Within the **Email Magic Links** drawer, toggle on **Enable the LoginOrCreate Flow**.
-     
-     <img width="400" alt="SDK Email Magic Links" src="https://user-images.githubusercontent.com/100632220/217053215-8c369de8-7828-4ad6-ac88-a50918520fc3.png">
-
-   - Toggle on **OAuth**.
-     
-     <img width="400" alt="SDK OAuth" src="https://user-images.githubusercontent.com/100632220/217053483-e757d1aa-af18-4af3-a476-45860ca3065f.png">
+2. Within your new Project, navigate to [SDK configuration](https://stytch.com/dashboard/sdk-configuration), and click **Enable SDK**.
 
-3. Navigate to [Redirect URLs](https://stytch.com/dashboard/redirect-urls), and add `http://localhost:3000/authenticate` as the types **Login** and **Sign-up**.
-   
-   <img width="400" alt="Redirect URLs" src="https://user-images.githubusercontent.com/100632220/217983021-d8bf6fff-6a68-4e94-bffd-d062e69c8817.png">
+3. Navigate to [OAuth](https://stytch.com/dashboard/oauth), and set up login for Google in the Test environment.
 
-
-4. Navigate to [OAuth](https://stytch.com/dashboard/oauth) and set up Google login in the Test environment. You don't need to copy the Google start URL for the Test environment because the Test environment is pre-configured to work! However, if you want to set up Google Oauth in the Live environment, follow all of the instructions provided in the Dashboard. If you are not interested in OAuth login you can skip this step. However, the _Continue with Google_ button in this application will not work in the Live environment.
-   
    <img width="400" alt="OAuth configuration" src="https://user-images.githubusercontent.com/100632220/217055674-a7dafc17-6ad3-492f-8dd2-92560d60dc00.png">
 
-
-5. Finally, navigate to [API Keys](https://stytch.com/dashboard/api-keys). You will need the `project_id`, `secret`, and `public_token` values found on this page later on.
+4. Finally, navigate to [API Keys](https://stytch.com/dashboard/api-keys). You will need the `project_id`, `secret`, and `public_token` values found on this page later on.
 
 ### On your machine
 
 In your terminal clone the project and install dependencies:
 
 ```bash
-git clone https://github.com/cal-stytch/test-stytch-nextjs-example.git
-cd test-stytch-nextjs-example
-npm i
+git clone https://github.com/cal-stytch/stytch-nextjs-example.git
+cd stytch-nextjs-example
+pnpm i
```

Comment:
Do enough people use `pnpm` that we should default to that instead of `npm`?

Reply:
This was a recommendation from Vercel's reviewing engineer. 

I'm certain that npm has a much larger user base, but Next.js would like you to use pnpm. 

It does. seem worthwhile to mention alternatives though, I will add that note.